templates/takeovers/_private_cloud_economics.html

```diff
@@ -0,0 +1,17 @@
+<section class="p-strip is-deep p-takeover--private-cloud-economics">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1 class="p-takeover__title">Busting the myth of private cloud economics</h1>
+      <p class="p-takeover__text">Analyst firm 451 Research, highlights TCO savings using BootStack, Canonical&rsquo;s private managed cloud service. </p>
+      <p class="u-hide--medium u-hide--large">
```

Comment:
`u-align--center` here to match mobile design

Reply:
Good catch